src/util/util.js

```diff
@@ -0,0 +1,11 @@
+export function checkKey(eventKey, key) {
+  if (eventKey === key) {
+    return true;
+  }
+
+  return false;
+}
```

Comment:
```suggestion
export function checkKey(eventKey, key) {
  return eventKey === key
}
```
이렇게 간단히도 작성할 수 있을 것 같아요 😄